app/validators/iban_validator.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class IbanValidator < ActiveModel::Validator
+  def validate(record)
+    return if record.iban.nil? || IBANTools::IBAN.valid?(record.iban)
```

Comment:
Gaat dit goed als er spaties in de string zitten? (Ik hoop van wel). Anders moeten we de spaties ergens strippen.

Reply:
goed punt. Ik heb een spec toegevoegd. Het gaat idd wel goed.